packages/remirror__extension-file/src/file-extension.tsx

```diff
@@ -219,6 +222,34 @@ export class FileExtension extends NodeExtension<FileOptions> {
     };
   }
 
+  @keyBinding({ shortcut: ['Backspace', 'Delete'] })
+  backspaceKey(props: KeyBindingProps): boolean {
+    const { tr, state } = props;
+    const { from, to, empty } = tr.selection;
+
+    if (!this.options.onDeleteFile || empty) {
```

Comment:
Sadly, `this.options.onDeleteFile` will always exist. It in fact points to [this inline function](https://github.com/remirror/remirror/blob/47df75996e26b9e8fc9b070e5444494605321610/packages/remirror__core/src/extension/base-class.ts#L377-L394). Since handlers are dynamic, you can add more handlers after the extension object is constructed. We might want a `hasHandler(key: string): boolean` in `@remirror/core` but I'm not sure.